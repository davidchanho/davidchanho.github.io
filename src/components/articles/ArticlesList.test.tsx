import React from "react";
import renderer from "react-test-renderer";
import ArticlesList from ".";

describe("ArticlesList", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ArticlesList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
