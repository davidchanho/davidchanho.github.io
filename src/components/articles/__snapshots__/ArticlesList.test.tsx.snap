// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticlesList renders correctly 1`] = `
<div
  className="list-group"
  onKeyDown={[Function]}
>
  <div
    className="list-group-item"
    data-rb-event-key={null}
    disabled={false}
    onClick={[Function]}
  />
</div>
`;
